docs/feed/feed.xml

```diff
@@ -1,25 +1,61 @@
 <?xml version="1.0" encoding="utf-8"?>
 <feed xmlns="http://www.w3.org/2005/Atom">
-	<title>Dat Project Blog</title>
-	<subtitle>New developments and ideas from the Dat team.</subtitle>
+	<title>Dat Ecosystem Blog</title>
+	<subtitle>New developments and ideas from the Dat Ecosystem.</subtitle>
 	
 	<link href="https://blog.datproject.org/feed/feed.xml" rel="self"/>
 	<link href="https://blog.datproject.org/"/>
-	<updated>2020-08-22T10:00:00Z</updated>
+	<updated>2021-12-21T10:00:00Z</updated>
 	<id>https://blog.datproject.org/</id>
 	<author>
 		<name>Dat Consortium</name>
 		<email>dat-consortium@protonmail.com</email>
 	</author>
 	
+	<entry>
+		<title>Dat Ecosystem Relaunch
+</title>
+		<link href="https://blog.datproject.org/dat-ecosystem-relaunch/"/>
+		<updated>2021-12-21T10:00:00Z</updated>
+		<id>https://blog.datproject.org/dat-ecosystem-relaunch/</id>
```

Comment:
i think the URL here is outdated.
Maybe thats a problem that occures in many places.
Is that easy to fix?

Reply:
Where do we link to? `https://blog.dat-ecosystem.org/` right?